source/validate.h

```diff
@@ -141,6 +141,10 @@ spv_result_t CfgPass(ValidationState_t& _,
 /// Performs Id and SSA validation of a module
 spv_result_t IdPass(ValidationState_t& _, const spv_parsed_instruction_t* inst);
 
+/// Performs data rules validation.
```

Comment:
E.g. "Performs validation of the Data Rules subsection of 2.16.1 Universal Validation Rules".
It might even merit saying that it's work in progress. ? (That can get stale.)